src/groups/mqb/mqba/mqba_domainmanager.cpp

```diff
@@ -702,6 +702,124 @@ int DomainManager::processCommand(mqbcmd::DomainsResult*        result,
             return 0;  // RETURN
         }
     }
+    else if (command.isRemoveValue()) {
+        const bsl::string& name = command.remove().domain();
+
+        DomainSp domainSp;
+
+        if (0 != locateOrCreateDomain(&domainSp, name)) {
+            bmqu::MemOutStream os;
+            os << "Trying to remove a nonexistent domain '" << name << "'";
+            result->makeError().message() = os.str();
+            return -1;  // RETURN
+        }
+
+        // First pass
+        if (command.remove().finalize().isNull()) {
+            BALL_LOG_INFO << "[First pass] DOMAINS REMOVE '" << name
+                          << "' called!!!";
+
+            // TODO: broadcast to all other nodes
+
+            // 1. Reject if there's any opened queue
+            if (domainSp->hasActiveQueue()) {
+                bmqu::MemOutStream os;
+                os << "Trying to remove the domain '" << name
+                   << "' while there are queues open";
+                result->makeError().message() = os.str();
+                return -1;  // RETURN
+            }
+
+            BALL_LOG_INFO << "BEFORE CHEKCING CLUSTER STATUS";
```

Comment:
Not sure we need these logs either, they look like they were just for debugging.